app/src/playStore/java/org/fossasia/susi/ai/rest/clients/YouTubeService.java

```diff
@@ -0,0 +1,5 @@
+package org.fossasia.susi.ai.rest.clients;
+
+public class YouTubeService {
+    public static final String API_KEY = "AIzaSyDR6eETvA9Pfz_6nntGUVcvqT8Lb9Gg4Ao";
```

Comment:
Bad idea! Move to Build Config and make it optional so that app doesn't crash if API Key is not found

Reply:
Field won't be optional, it'll be a config parameter loaded from an environment variable. This needs to handled in such a way that if the env var is unavailable, the app does not crash